basilisk/gui/main_frame.py

```
@@ -656,9 +650,9 @@ def on_add_attachments(
 			)
 			return
 		if from_url:
-			current_tab.add_attachment_url_dlg(event)
+			current_tab.prompt_panel.add_attachment_url_dlg(event)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 3 spaces, expected 12

(W0311)